ui/__tests__/components/node-renderers/math.test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import TeXMath from '../../../components/node-renderers/math';
+import {
+  itIncludesTheIdentifier,
+} from '../../test-utils/node-renderers';
+
+jest.mock('../../../util/render-node');
+
+describe('<TeXMath />', () => {
+  itIncludesTheIdentifier(TeXMath, { text: 'abcd' });
+  it('generates something math-like', () => {
+    const docNode = {
+      identifier: 'aaa_1__bbb_2__ccc_3_math',
+      text: '\\frac{10}{11}',
+    };
+    const result = mount(<TeXMath docNode={docNode} />);
+    expect(result.render().find('span.katex')).toHaveLength(1);
+    expect(result.html()).toContain('10');
+    expect(result.html()).toContain('11');
```

Comment:
I see that these numbers are coming from line 15, but I wonder if there's not a risk that they're also part of the generated css classes? Would it make sense to use `result.text()` instead? Or maybe less "common" input?

Reply:
You're right, I should change these to rarer numbers. (I'll also test the text option.)